client/src/config.tsx

```diff
@@ -192,13 +190,13 @@ export const appQuestions: Step[] = [
       },
       {
         type: 'file',
-        id: 'resume',
+        id: 'resumeLink',
         question: <>Upload your resume in PDF format below (Max: 100MB).</>,
```

Comment:
think max is 2MB

also: do we do any client-side validation? Let's tell them it's .pdf, .docx, .doc only and hopefully be able to check on the client for them as well